tests/lib/pkgdb.sh

```diff
@@ -31,13 +31,24 @@ fedora_name_package() {
     esac
 }
 
+opensuse_name_package() {
```

Comment:
It just looks odd as nothing else uses this style. If anything I'd call it opensuse_package_name (`object.object.object` style)